packages/core/src/modules/connections/messages/ConnectionInvitationMessage.ts

```diff
@@ -107,7 +109,7 @@ export class ConnectionInvitationMessage extends AgentMessage {
    * @param domain domain name to use for invitation url
    * @returns invitation url with base64 encoded invitation
    */
-  public toUrl({ domain, useLegacyDidSovPrefix = false }: { domain: string; useLegacyDidSovPrefix?: boolean }) {
+  public toUrl({ domain, useLegacyDidSovPrefix }: { domain: string; useLegacyDidSovPrefix?: boolean }) {
```

Comment:
I was tinkering a bit and forgot to set this back to false. It doesn't really matter as it will only use legacy if the value is true (so defalut undefined or false doesn't matter).

I've reverted the change